src/errors/test-run/templates.js

```diff
@@ -207,5 +207,9 @@ export default {
         Action "${err.selectorName}" argument error:
 
         ${escapeHtml(err.errMsg)}
+    `),
+
+    [TYPE.cantObtainInfoForElementSpecifiedBySelectorError]: err => markup(err, `
+        Can't obtain information for node because specified selector does not match any node in the DOM tree.
```

Comment:
Yep, I thought that it's just Armina who doesn't like contractions but MS Word also treats all contractions as "poor style" - so yes.

```
        Cannot obtain information about the node because the specified selector does not match any node in the DOM tree.
```